services/security/src/integration-tests/bootstrap.ts

```diff
@@ -48,12 +50,8 @@ function getEntities(connection: Connection): entity[] {
 
 async function cleanAll(connection: Connection, entities: entity[]) {
   try {
-    const actionsToCleanRepositories = entities.map((entity) => {
-      const repository = connection.getRepository(entity.name);
-      return repository.query(`TRUNCATE TABLE "${entity.tableName}" CASCADE;`);
-    });
-
-    return Promise.all(actionsToCleanRepositories);
+    const query = `TRUNCATE TABLE ${entities.map((entity: any) => `"${entity.tableName}"`).join(",")} CASCADE;`;
+    return connection.query(query);
   } catch (error) {
     throw new Error(`ERROR: Cleaning test db: ${error}`);
```

Comment:
This Error will never thrown

Reply:
You are right 👍  
I have removed `try catch`